src/XrdCeph/XrdCephOss.cc

```diff
@@ -46,6 +46,42 @@ XrdVERSIONINFO(XrdOssGetStorageSystem, XrdCephOss);
 XrdSysError XrdCephEroute(0);
 XrdOucTrace XrdCephTrace(&XrdCephEroute);
 
+bool allCharsAlnum(std::string candidate) {
```

Comment:
would suggest to pass the input by const reference (rather than by value).

Reply:
I should be more aware of const-ness in the future! However, I've now deleted th allCharsAlnum and sanitizePath functions and reverted to the translateFileName logic, copied over from the XrdCephPosix layer.